pkg/solana/config/config.go

```diff
@@ -42,6 +42,9 @@ var defaultConfigSet = Chain{
 
 	// log poller
 	LogPollerStartingLookback: config.MustNewDuration(24 * time.Hour),
+	LogPollerMaxRetries:       ptr(uint8(18)), // max number of retries for log poller
```

Comment:
MaxRetries only controls how many times the job should fail before we log critical error. I'm not sure that we should increase it. The job won't be removed from the queue until it succeeds.    

AFAIK backoff is caluclated calculated as follows: 
```
return time.Duration(2<<retries) * 100 * time.Millisecond
```
So the delay before 18th  retry  may take up to 873 minutes, if my math is correct.